src/vm/parser.rs

```diff
@@ -76,7 +77,7 @@ pub fn lex(input: &str) -> Result<Vec<(LexItem, u32, u32)>> {
         LexMatcher::new("'CT(?P<value>[0123456789ABCDEFGHJKMNPQRSTVWXYZ]{28,41}.([[:alpha:]]|[-]){5,40})", TokenType::QualifiedContractPrincipalLiteral),
         LexMatcher::new("'CT(?P<value>([[:alpha:]]|[-]){5,40})", TokenType::ContractPrincipalLiteral),
         LexMatcher::new("'(?P<value>[0123456789ABCDEFGHJKMNPQRSTVWXYZ]{28,41})", TokenType::PrincipalLiteral),
-        LexMatcher::new("(?P<value>([[:word:]]|[-#!?+<>=/*])+)", TokenType::Variable),
+        LexMatcher::new("(?P<value>([[:word:]]|[-!?+<>=/*])+)", TokenType::Variable),
```

Comment:
Aren't we supposed to escape `?`, `+` and  `*` ?
I did notice while working on #1078 that the `QualifiedContractPrincipalLiteral ` regex is not working as expected, because of the "." not being escaped.

Reply:
inside of regex brackets (`[` and `]`), those characters shouldn't be escaped.

Reply:
Though you're right, that outside of the brackets, those should be escaped (which is why the `.` needs to be escaped in the `QualifiedContractPrincipalLiteral`)